src/mscorlib/src/System/Array.cs

```diff
@@ -334,6 +334,19 @@ public static void Copy(Array sourceArray, long sourceIndex, Array destinationAr
             Array.Copy(sourceArray, (int) sourceIndex, destinationArray, (int) destinationIndex, (int) length);
         }
 
+        [ReliabilityContract(Consistency.MayCorruptInstance, Cer.MayFail)]
+        public static void Copy<T>(T[] sourceArray, T[] destinationArray, int length)
```

Comment:
The generic method won't be always precompiled by crossgen. It would be also useful to understand how much extra JITing is this going to introduce in real workloads on shipping config that has a bunch of stuff crossgened.

cc @lt72